africanlii/views/documents.py

```diff
@@ -56,5 +56,12 @@ def render_to_response(self, context, **response_kwargs):
                 file = open(f"{temp_dir.name}/{filename}", "rb")
             else:
                 file = self.object.source_file.file.open()
-            return view_attachment(file)
+            response = FileResponse(
```

Comment:
You're reverting your change from https://github.com/laws-africa/peachjam/pull/208 -- we can't use FileResponse. This needs to be a normal HttpResponse, very much like how `view_attachment` does it above.